src/App.css

```diff
@@ -6,16 +6,20 @@
   width: 100%;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
   font-size: calc(3px + 2vmin);
   color: rgb(37, 3, 3);
   overflow: hidden;
 }
 
+@media (min-width: 819px) {
+  .app {
+    justify-content: center;
+  }
+}
```

Comment:
Please don't hardcode breakpoints like this. Check material UI. They have well researched breakpoints. Use them plz. The closest breakpoint that mui have to `819px` is `900px`. Use that. My suggestion would be to remove this `.app` class entirely. And instead of `div` use a mui `Box`. And using mui styling system you can give it styling according to breakpoints.